docs/devel/api_changes.md

```diff
@@ -485,6 +485,17 @@ To regenerate them:
 hack/update-generated-deep-copies.sh
 ```
 
+Since our components still use internal apis, you may need to generate code
```

Comment:
The migration for conversions is now finished. Now generation happens with the same update-codegen.sh script.
Can you please update this PR then?